.gitignore

```diff
@@ -1,5 +1,7 @@
 data/*
 example_data/*
+src/run_new_garnet.py
```

Comment:
Why this?

Reply:
not sure what that is

Reply:
I guess I'm asking "what is `run_new_garnet.py`?

Reply:
No idea. It has some paths from Divya's folder...removed it.